generator/common/validations.py

```diff
@@ -0,0 +1,31 @@
+"""Validations
+
+This file provides the support of basic validations for unit tests
+
+"""
+import pandas as pd
+
+
+def validate_csv(filename, header, cols, rows):
+    """
+    Validates a files data using the provided parameters
+
+    :param file: File and path to be validated
```

Comment:
```suggestion
    :param filename: File and path to be validated
```

Reply:
Thanks Timothy, updated this in the commit below